inc/consent/namespace.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Altis\Core\Consent;
+
+use Altis;
+use Altis\Consent\Settings;
+
+/**
+ * Kick it off.
+ */
+function bootstrap() {
+	add_action( 'plugins_loaded', __NAMESPACE__ . '\\set_consent_options' );
+	add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_plugins', 1 );
+
+	add_filter( 'altis.analytics.noop', __NAMESPACE__ . '\\set_analytics_noop' );
+	add_filter( 'altis.analytics.data', __NAMESPACE__ . '\\set_analytics_data', 1 );
+
+	// If statistics hasn't been consented to, don't load the GTM output.
+	if ( ! wp_get_consent( 'statistics' ) ) {
+		remove_action( 'wp_head', '\\HM\\GTM\\output_tag', 1, 0 );
+		remove_action( 'after_body', '\\HM\\GTM\\output_tag', 1, 0 );
+	}
+}
+
+/**
+ * Save the defaults to the database if nothing has been set yet.
+ */
+function set_consent_options() {
+	$config  = Altis\get_config()['modules']['core']['consent'];
+	$fields  = Settings\get_cookie_consent_settings_fields();
+	$options = [];
+
+	// Bail if we've turned consent off explicitly.
+	if ( empty( $config ) ) {
+		return;
+	}
+
+	// Check if display-banner was configured.
+	if ( ! empty( $config['display-banner'] ) ) {
+		// Make sure display-banner is a boolean.
+		if ( is_bool( $config['display-banner'] ) ) {
+			$options['display_banner'] = $config['display-banner'];
+			unset( $fields['display_banner'] );
+		}
+	}
+
+	// Check if banner-options was configured.
+	if ( ! empty( $config['banner_options'] ) ) {
+		// Make sure any option set in the config is a valid option.
+		if ( in_array( $config['banner-options'], Settings\get_cookie_banner_options(), true ) ) {
```

Comment:
This compares the string in `banner-options` with the deeper `<select>`-style options array returned from this function, so doesn't match.
```suggestion
		if ( in_array( $config['banner-options'], wp_list_pluck( Settings\get_cookie_banner_options(), 'value' ), true ) ) {
```